finding/finding_test.go

```diff
@@ -184,7 +167,12 @@ func Test_New(t *testing.T) {
 		t.Run(tt.name, func(t *testing.T) {
 			t.Parallel()
 
-			r, err := New(testfs, tt.id)
+			content, err := os.ReadFile(tt.path)
+			if err != nil {
+				panic(err)
```

Comment:
Lets avoid panic'ing, how about `t.Fatalf`

Reply:
fine with me. What's the difference?

Reply:
they both cause the test to fail with a message, but panic clogs up the output with the goroutine traces.

Reply:
thanks. will keep in mind.